lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -171,13 +164,16 @@ class ImageUsage extends Gatherer {
 
     const imageUsage = [];
     for (let element of elements) {
-      // link up the image with its network record
-      element.networkRecord = indexedNetworkRecords[element.src];
+      // Pull some of our information directly off the network record.
+      const networkRecord = indexedNetworkRecords[element.src] || {};
+      element.mimeType = networkRecord.mimeType;
+      const {resourceSize = 0, transferSize = 0} = networkRecord;
+      element.resourceSize = Math.min(resourceSize, transferSize);
```

Comment:
we didn't have one before, but mind putting a comment on this about the goal/why it's ok to fall back to `transferSize`?

Reply:
sure, done